design/async-commit/spec.md

```diff
@@ -68,15 +73,21 @@ message LockInfo {
 
 Implemented in TiDB, TiSpark.
 
-The client has system variables for supporting async commit (`tidb_enable_async_commit`), supporting 1pc (`tidb_enable_1pc`), and forcing external consistency when using async commit (`tidb_guarantee_external_consistency`). The client has configuration options for setting the limits on the size (`tikv-client.async-commit.total-key-size-limit`) and number of keys (`tikv-client.async-commit.keys-limit`) in an async commit transaction. When async commit is enabled, a transaction will not use async commit if it exceeds either of the size limits.
+The client has system variables for supporting async commit (`tidb_enable_async_commit`), supporting 1pc (`tidb_enable_1pc`), and forcing external consistency when using async commit (`tidb_guarantee_external_consistency`). The client has configuration options for setting the limits on the size (`tikv-client.async-commit.total-key-size-limit`) and number of keys (`tikv-client.async-commit.keys-limit`) in an async commit transaction. When async commit is enabled, a transaction will not use async commit if it exceeds either of the size limits. The rest of the spec assumes that async commit or 1pc are enabled.
```

Comment:
Should we describe the exact variables in TiDB? A common KV client may not have "system variables".

Reply:
I think it is useful to have some detail about TiDB (and TiFlash, etc.) Writing for a generic client is more 'pure', but I think  using TiDB as the main client is more practical